.circleci/config.yml

```
@@ -0,0 +1,89 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+commands:
+  circleci_configure_path:
```

Let's consistently give these sections headings.

```suggestion
  ##################
  # CircleCI commands
  ##################
  circleci_configure_path:
```